unittests/resources/checks_unlisted/good.py

```diff
@@ -31,6 +32,7 @@ def __repr__(self):
 
 
 @rfm.parameterized_test(*({'a': x, 'b': y} for x in range(3) for y in range(2)))
+@rfm.extend_test
```

Comment:
I guess this is needed for the `__eq__()`, right? That's why we need to mark final only specific framework methods.